experiments/axon_geometry/scripts/download_segments.py

```diff
@@ -0,0 +1,43 @@
+import boto3
+from botocore import UNSIGNED
+from botocore.client import Config
+import os
+from pathlib import Path
+
+cwd = Path(os.getcwd())
+exp_dir = cwd.parents[0]
```

Comment:
When I run this, it downloads the data to experiments/data rather than experiments/axon_geometry/data. Can you fix this?

Reply:
Fixed here: https://github.com/NeuroDataDesign/brainlit/blob/012ca4f332e4920de29cfca31a86dd8835815e75/experiments/axon_geometry/scripts/download_segments.py#L7-L10